src/animations.js

```diff
@@ -38,10 +99,31 @@ const ANIMATIONS = {
 }
 
 const CURRENT_ANIMATION_INFO = {
+    karaokeState: null,
     name: null,
     timeScrolledIntoView: null
 }
 
+let FAILED_KARAOKE_UPDATE_ATTEMPTS = 0
+
+
+/**
+ * Even with the correct import order and with the `defer` attribute applied, `updateKaraoke` may not be ready to be
+ * invoked by the time the animation starts.
+ */
+function attemptUpdateKaraoke(karaokeAnimationInfo, animationTime) {
+    if (typeof updateKaraoke !== 'undefined') {
+        updateKaraoke(karaokeAnimationInfo, animationTime)
+    } else {
+        FAILED_KARAOKE_UPDATE_ATTEMPTS++
+        // Assuming the animation runs at 60 FPS, this will give about a second of cushion before sending a Sentry
+        // error.
+        if (FAILED_KARAOKE_UPDATE_ATTEMPTS === 60) {
```

Comment:
Since it still happened 3 times out of a few dozens, I feel we should either:
- Increase the threshold to make sure it only alerts if something is clearly broken
- Set the alert degree to warning, that we can then monitor in Sentry